src/riscv-hybrid-integration.adoc

```diff
@@ -279,9 +279,20 @@ shown in xref:default-csrnames-added[xrefstyle=short].
 
 {cheri_default_ext_name} optionally allows <<MODESW>> to execute in debug mode.
 
-When entering debug mode, the core always enters {cheri_cap_mode_name}.
+When entering debug mode, whether the core enters {cheri_int_mode_name} or
+{cheri_cap_mode_name} is UNSPECIFIED.
```

Comment:
This text with the UNSPECIFIED is out-of-data. It is specified now.

Reply:
The text at https://github.com/riscv/riscv-cheri/pull/370/files#diff-b366fcf81637594697f776b0c57bf43a576436df8b1fde7ed1ae593771025d94R92 is out of date. The debug mode is given by dinfc and the code sequence must us gcmode instead of gctag.

Reply:
fixed.